### PR TITLE
Added workshop page and updated ticket page plus navbar

### PR DESCRIPTION
**Pull request by thatdocslady**
My local staging is borked so hopefully this will pass all the checks here.
Still need to write the blog post but I figured it's better to get the official web content in first and then write the announcement.

### CODE REVIEW

File: docs/conf/na/2017/workshops.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+This year we are launching workshops, a new activity type inspired once again by our friends at various open-source software conferences. The workshops will be held during the writing day, in dedicated rooms in the conference venue.
+
+The main goal of the workshops is to bring together folks who are interested to learn from different experts in relevant fields in a focused environment.
+
+What can I learn in the workshops?
+----------------------------------
+
+This year we have 2 workshops that cover one writing-related topic and one development-related topic. This is a part of our efforts to cross-train the different roles that our community members encompass.
+
+For specific information about this year's topics, see the `Sessions page <http://www.writethedocs.org/conf/na/2017/speakers/>`_. The workshop sessions are labeled with a "Workshop" prefix.
```
Comment (ericholscher) [inline docs/conf/na/2017/workshops.rst]: We could also link directly to each talk, since we only have 2.

**Review by ericholscher — COMMENTED** [review]
Looks great overall :+1:

File: docs/conf/na/2017/workshops.rst [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+=========
+
+This year we are launching workshops, a new activity type inspired once again by our friends at various open-source software conferences. The workshops will be held during the writing day, in dedicated rooms in the conference venue.
+
+The main goal of the workshops is to bring together folks who are interested to learn from different experts in relevant fields in a focused environment.
+
+What can I learn in the workshops?
+----------------------------------
+
+This year we have 2 workshops that cover one writing-related topic and one development-related topic. This is a part of our efforts to cross-train the different roles that our community members encompass.
```
Comment (plaindocs) [inline docs/conf/na/2017/workshops.rst]: > This year we have two workshops, one that covers writing related topics and one that covers development related topics.

^ reads better to me. Also, calling git development related is a stretch I think. Development AND docs as code? :-p :-p

File: docs/conf/na/2017/workshops.rst [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+Based on feedback and demand, we might increase the number of workshops in the following years. Please let us know how we can develop this activitiy!
+
+Is the workshop included in my conference ticket?
+-------------------------------------------------
+
+In short, no. Due to the amount of work required by the instructors to prepare the workshops, and the attention that is needed to be given during the workshops, these are priced separately.
+
+We created individual tickets on our `registration website <https://ti.to/writethedocs/write-the-docs-na-2017/>`_ so that you can register to whichever workshop you choose separately.
+
+The workshops have a limited capacity to support an effective and comfortable learning experience.
```
Comment (plaindocs) [inline docs/conf/na/2017/workshops.rst]: > he workshops have a limited capacity OF X PEOPLE, so we can provide an effective and comfortable learning experience.

maybe?

File: docs/conf/na/2017/workshops.rst [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+What should I do to prepare for the workshop?
+---------------------------------------------
+
+Get excited about learning a new thing! Of course, bringing a laptop or writing tools to take notes is usually recommended.
+
+Some workshops might require a laptop with some software prerequisites; those will be communicated to the workshop participants by email closer to the workshop date.
```
Comment (plaindocs) [inline docs/conf/na/2017/workshops.rst]: Some when we have two is odd. Could just say "participants will receive an email listing any (software) requirements well in advance 

Also, as workshops I think we might be better saying 'bring a laptop'

**Review by plaindocs — COMMENTED** [review]
Looks good. Couple of nitpicks.

**Comment by ericholscher** [comment]
Any update on this? Should get it out.

**Comment by ericholscher** [comment]
Went ahead and fixed it up and pushed it -- feel free to edit more if you want.

**Pull request merged**